nipype/utils/config.py

```diff
@@ -91,7 +91,7 @@ def __init__(self, *args, **kwargs):
         self._config = configparser.ConfigParser()
         self._cwd = None
 
-        config_dir = os.path.expanduser("~/.nipype")
+        config_dir = os.path.expanduser(os.getenv("NIPYPECONFDIR", default="~/.nipype"))
```

Comment:
How do you feel about:

```suggestion
        config_dir = os.path.expanduser(os.getenv("NIPYPE_CONFIG_DIR", default="~/.nipype"))
```